demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -366,14 +366,32 @@ def is_pack_metadata_desc_too_long(self, description_name):
                 return False
         return True
 
+    def validate_support_details_exist(self, pack_meta_file_content):
+        """Validate either email or url exist in contributed pack details."""
+        if not pack_meta_file_content[PACK_METADATA_URL] and not pack_meta_file_content[PACK_METADATA_EMAIL]:
+            if self._add_error(Errors.pack_metadata_missing_url_and_email(), self.pack_meta_file):
+                return False
+
+        return True
+
+    def validate_metadata_url(self, pack_meta_file_content):
+        """Validate the url in the pack metadata doesn't lead to a github repository."""
+        metadata_url = pack_meta_file_content[PACK_METADATA_URL]
+        metadata_url = metadata_url.lower().strip()
+        if len(re.findall("github.com", metadata_url)) > 0:
+            if not metadata_url.endswith('/issues'):
```

Comment:
please add a dev comment about this. Also might be wise to note this loophole in the error message itself.